cfme/v2v/migration_plans.py

```diff
@@ -291,15 +291,19 @@ def after_fill(self, was_change):
     @View.nested
     class schedule(View):  # noqa
         create = Button("Create")
+        select_migration = RadioGroup('.//div[contains(@id,"migration_plan_type_radio")]')
```

Comment:
@sshveta  The above BTN is gave an option to select either cold or warm migration, So "select_migration" seems more appropriate name for this BTN, By default it point to cold migration and for warm migration , we are using this BTN to select warm migration , we are using "start_selection = "Warm Migration ---" down the line.

Reply:
@mnadeem92 : The radio button with Id `migration_plan_type_radio` points to cold migration button and button with `id=migration_plan_choice_radio` points to warm_migration . Earlier when only cold_migration was the option, select_migration was okay but now buttons with name 'select_migration' for cold and 'run_migration` for warm doesn't make sense . I suggest we have their names clear so that we know which type we are choosing

Reply:
@sshveta It will be clear to you if you visit the select migration page. Let me elaborate it.
There is 2 section
1.  select_migration = RadioGroup('.//div[contains(@id,"migration_plan_type_radio")]')  --> will point to 2 radio button  --> a. Cold Migration(by default select) b. Warm Migration

Now, depending on selection of any one of these, below option gets visible in 2nd half of the page
 1. If  Warm Migration Selected in above case--> Then a plain text appear (See screen shot)
2. If Cold Migration Select  ---> Again 2 radio button appears , either to run migration immediately or schedule it for later --> These radio button was named as run_migration BTN from the start which i thought make sense, as for warm migration we so not need to run migration, instead we need to do cutover in later stage of migration after pre-copy gets completed and for the same we have written the schedule_cutover method.  Please find the screen shots.